src/main/resources/idprovider/idprovider.js

```diff
@@ -15,7 +27,11 @@ exports.handle401 = function() {
     };
 };
 
-exports.get = function() {
+exports.get = function (req) {
+    if (req.rawPath.indexOf('/_static/') !== -1) {
```

Comment:
it should start with `/_static/`, to be static resource. not just contain `/_static/`